### DISCUSSION TITLE
Excute the operation using GitHub Actions

### DISCUSSION THREAD

**Discussion by ghost** [Schwartzblat/WhatsAppPatcher#31] [Ideas]
Hi, why don't you execute your work using GitHub Actions to offer more transparency and also it will be easier that you don't have to upload the apk file everytime you publish a new release, me and many people like to compile everything themselves but I don't have a PC at this moment to do the work, I think it is very easy to accomplish that even you won't need to do the work, you only need to fix the scripts or launch the workflow from now and then, and we can get updates directly from your repo using [Obtainium](https://github.com/ImranR98/Obtainium), I actually have a good example if you take a look at my [Signal Fork repository](https://github.com/JohnBetaro/Signal-Android-AMOLED-Dark-Plus), I don't have to do anything, everything is handled by GitHub Actions, a good example also is [Revanced Extended](https://github.com/NoName-exe/revanced-extended).

Thank you for the awesome work.

**Comment by wt383** [comment]
wow this is a game changer! i hope they do this. ty for the info.

**Reply by ghost** [reply]
Already done in the latest release.
